primitives/storage/src/lib.rs

```diff
@@ -232,6 +232,9 @@ pub mod well_known_keys {
 	}
 }
 
+/// Default value to use as a threshold for inner hashing.
+pub const DEFAULT_MAX_INLINE_VALUE: u32 = 33;
```

Comment:
So does this mean that values of size 33 still end up inlined and 34 gets moved to a separate node?

Reply:
32 inlined and 33 in a separate node. This is the value applied for all V1 value, could be set higher, not really lower.

Makes me think `DEFAULT_MIN_VALUE_NODE_LEN` would be a better naming.
Would need similar renaming on the trie PR (max_inline_value -> min_inline_node).
Or could also make it a inclusive max value.